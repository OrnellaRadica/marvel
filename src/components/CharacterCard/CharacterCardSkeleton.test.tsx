import renderer from "react-test-renderer";
import { CharacterCardSkeleton } from ".";

it("Should render the skeleton card", () => {
  const characterCard = renderer.create(<CharacterCardSkeleton />).toJSON();
  expect(characterCard).toMatchSnapshot();
});
