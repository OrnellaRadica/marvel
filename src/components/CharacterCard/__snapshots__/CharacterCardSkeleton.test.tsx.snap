// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the skeleton card 1`] = `
<div
  className="sc-ead767e7-0 jeLVew"
  data-testid="skeleton-card"
>
  <div
    className="sc-ead767e7-1 gARGUb"
  />
  <div
    className="sc-ead767e7-2 hPYKlF"
  >
    <div
      className="sc-ead767e7-3 lnUtYP"
    />
    <div
      className="sc-ead767e7-4 gAfFr"
    />
    <div
      className="sc-ead767e7-5 fScbDl"
    />
  </div>
</div>
`;
